File: kevoree-library/javase/org.kevoree.library.javase.bluetooth/src/main/scala/org/kevoree/channels/Tester2.scala
package org.kevoree.channels

import javax.bluetooth.LocalDevice
import com.intel.bluetooth.RemoteDeviceHelper
import javax.microedition.io.{StreamConnection, Connector}
import java.io.{ObjectOutputStream, PrintStream}
import org.kevoree.framework.message.Message

/**
 * Created by IntelliJ IDEA.
 * User: duke
 * Date: 23/09/11
 * Time: 16:55
 * To change this template use File | Settings | File Templates.
 */

object Tester2 extends App {

  val discovery = new BluetoothDiscovery

  discovery.getRemoteDevice("POWET") match {
    case Some(device)=> {

      val conn = Connector.open("btspp://"+device.getBluetoothAddress+":1");

      //  val sender = new PrintStream(conn.asInstanceOf[StreamConnection].openOutputStream())
      // sender.println("hello")

      val    out = new ObjectOutputStream(conn.asInstanceOf[StreamConnection].openOutputStream());

      out.writeObject(new Message());
      conn.close()

    }
    case None => println("Not found")
  }


}

File: kevoree-core/org.kevoree.kcl/src/main/java/org/kevoree/kcl/Tester2.java
package org.kevoree.kcl;

/**
 * Created with IntelliJ IDEA.
 * User: jed
 * Date: 30/05/13
 * Time: 18:26
 * To change this template use File | Settings | File Templates.
 */
public class Tester2 {


    public static void main(String argv[]) throws Exception{

        KevoreeJarClassLoader kcl = new KevoreeJarClassLoader();
        kcl.add("/home/jed/.m2/repository/org/slf4j/slf4j-api/1.7.5/slf4j-api-1.7.5.jar");

        kcl.loadClass("org.slf4j.LoggerFactory");

        System.out.println("ici") ;
        kcl.add("/home/jed/.m2/repository/org/slf4j/slf4j-api/1.6.5/slf4j-api-1.6.5.jar");


        kcl.loadClass("org.slf4j.LoggerFactory");

    }
}
